handler.go

```diff
@@ -1981,6 +2016,126 @@ func (h *Handler) handlePostInput(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+// handlePostClusterResizeSetCoordinator handles POST /cluster/resize/set-coordinator request.
+func (h *Handler) handlePostClusterResizeSetCoordinator(w http.ResponseWriter, r *http.Request) {
```

Comment:
I'm 100% sure. We originally had this message contain both `oldCoordinator` and `newCoordinator` so that at the very least there was a confirmation by the node before changing. But that opened up the possibility of the cluster getting in a strange state as well. We should discuss this decision and perhaps propose other options.